### ISSUE TITLE
[NOWEB] - message.ack not received

### ISSUE THREAD

**Issue by ehussain** [allburov/whatsapp-http-api#165]
Hello,

I have created a session with following config.

```
[
  {
    "name": "XXXXX",
    "status": "WORKING",
    "config": {
      "proxy": null,
      "webhooks": [
        {
          "url": "xxxxxxxxxxxxxxxxxxxxxxx",
          "events": [
            "message",
            "message.any",
            "message.ack",
            "presence.update"
          ],
          "hmac": null,
          "retries": null,
          "customHeaders": null
        }
      ]
    }
  }
]
```

However we are not receiving any of the message.ack events for outgoing messages.

**Comment by allburov** [comment]
Hi! You should get a error message in docker logs, like this
```
Engine does not support webhook event: message.ack
```

I'll look at this, should be possible to add!

**Comment by allburov** [comment]
Hi! `message.ack` event is available in `2023.8.19` (Plus version).
We'll release it in Core and ARM versions in September, let me know if you want to have it early in ARM, I'll release it too!
